scripts/extract_nucelotide_info.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import json
+from collections import Counter, defaultdict
+import sys
+
+reference_path = sys.argv[1]
+
+with open(reference_path) as f:
+    reference_json = json.load(f)
+
+
+with open("scripts/output/nucleotide_info.csv", "w") as f_ni:
+    nucleotide_info = csv.writer(f_ni)
+    nucleotide_info.writerow(
+        [
+            "virus_name",
+            "abbr",
+            "name",
+            "iso_id",
+            "seg_id",
+            "A_percent",
+            "C_percent",
+            "T_percent",
+            "GC_percent",
+            "seg_len",
```

Comment:
This is an opportunity to fix this on our side of the "glue layer". This is kinda what I meant: fixing incorrect or unclear naming.
```suggestion
            "sequence_id",
            "virus_acronym",
            "virus_id",
            "virus_name",
            "isolate_id",
            "a_percent",
            "c_percent",
            "t_percent",
            "gc_percent",
            "length",
```